lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -32,12 +32,19 @@ namespace Js
         SimplePropertyDescriptor(BuiltInPropertyRecords::name, PropertyConfigurable)
     };
 
+    SimplePropertyDescriptor const JavascriptLibrary::FunctionWithLengthAndNameTypeDescriptors[2] =
```

Comment:
Its possible. We'll have to think about how many properties do we want to extend SimpleTypeHandler to. Perhaps @abchatra or @curtisman could comment?
(shouldn't be too bad to add a SimpleTypeHandler with 3 properties; still better than creating a dictionary type with 3 properties)